data/pages-content/visa.md

```diff
@@ -1,34 +1,38 @@
 ---
 
 # Visa Information
 
 ## Requirements
-If you want to attend EuroPython 2022 in Dublin, Ireland, you should first verify whether you need a visa to enter Ireland, and apply for one if you do.
+If you want to attend EuroPython 2023 in Prague, the Czech Republic, you should first verify whether you need a visa to enter the Czech Republic, and apply for one if you do.
 
-[This list of countries](https://irishimmigration.ie/wp-content/uploads/2021/07/Immigration-Service-Delivery-Visa-and-Non-Visa-Required-Countries.pdf) defines if you require a visa to visit Ireland.
+[This list of countries](https://www.mzv.cz/jnp/en/information_for_aliens/short_stay_visa/list_of_states_whose_citizens_are/index.html) defines if you require a visa to visit the Czech Republic.
 
-Note: Ireland is part of the EU, but it is NOT part of the Schengen area. So if you are from outside the EU and have a Schengen visa, you may still need to apply for an Irish visa. Please check the website above if you are uncertain.
+**NOTE**: The Czech Republic is part of the EU, and part of the Schengen Area. So if you already have a valid Schengen visa, you may **NOT** need to apply for an Czech visa. Please check the website above and consult your local consular office or embassy, if you are uncertain.
 
-If you do need a visa to attend EuroPython 2022, you can apply for the Irish [Short Stay (C) - Conference/Event Visa](https://www.irishimmigration.ie/coming-to-work-in-ireland/what-are-my-options-for-working-in-ireland/coming-to-work-for-less-than-90-days/conference-event-visa/) (Answer *reason for travel: ‘Conference/Event’*).
+If you do need a visa to attend EuroPython 2023, you can lodge a visa application issued for **[Short Stay (C), up to 90 days,  for the purpose of “Business /Conference"](https://www.mzv.cz/jnp/en/information_for_aliens/short_stay_visa/conference.html)**
 
-Please make sure to read the visa pages (linked above) carefully before making your application; the EuroPython organisers are not equipped to give any visa advice. If you do require a visa to travel to Ireland, it is advisable to submit your visa application at least **8 weeks** before your planned travel date.
+Please make sure to read the visa pages and carefully and prepare all the required documents, before making your application. the EuroPython organisers are not equipped to give any visa advice.
 
-## Visa Support Letters
-If you have a ticket to attend EuroPython 2022 and require a visa, you are welcome to apply for a support letter from the EuroPython Society. Simply fill in the form **within the 4 week period prior to the date of your visa application**:
+If you do require a visa to travel to the Czech Republic, it is advisable to submit your visa application **as soon as possible, but at least 15 days** before your planned travel date.
```

Comment:
This would differ from country to country. I would just stick with it as soon as possible.
For example, when I applied for my CZ visa in 2020, it took more than a month.

Reply:
I sort of wanted to give a more specific indication. How about something like:  "Even though it is still possible to submit your visa application 15 days prior to your travel, it is advisable to do so as soon as possible, as it can sometimes take more than a month to receive a decision"?

Reply:
Yes to:

> Even though it is still possible to submit your visa application 15 days prior to your travel, it is advisable to **apply as soon as possible**, as it can sometimes take more than a month to receive a decision.